### PR TITLE
Pre publish PR for new version of checkout-ui-extensions

### PR DESCRIPTION
**Pull request by jamesvidler**
### Background

This brings over the latest from `checkout-web`. Includes minor content updates and this [change to Text emphasis](https://github.com/Shopify/checkout-web/pull/14329).

Corresponding [Docs PR](https://github.com/Shopify/shopify-dev/pull/26644).

### Solution

Follow instructions [publish new versions of the packages](https://checkout--full-partners-setup-update-ii.docs.shopify.io/docs/referencedocs/web/extensions/libraries).

### 🎩
First, clone this repo and run `yarn && yarn build` to build the packages.
Then, you can tophat this package by scaffolding an extension with the CLI and run it using `yarn dev`. Then, manually copy and replace the `node_modules/@shopify/checkout-ui-extensions` and `node_modules/@shopify/checkout-ui-extensions-react` from this PR into the extension.

### Checklist

- [X] I have :tophat:'d these changes
- [X] I have updated relevant documentation - https://github.com/Shopify/shopify-dev/pull/26644

### CODE REVIEW

File: packages/checkout-ui-extensions/src/components/Text/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ optional = ?
...
+| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic. Combine with an accessibilyRole of 'offset' or 'stress' to add more meaning to the text.<br /><br /><code>"bold"</code>: Set the text in bold. |
```
Comment (jffortier) [inline packages/checkout-ui-extensions/src/components/Text/README.md:12]:
```suggestion
| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic. Combine with an “accessibilyRole“ of “offset“ or “stress“ to add more meaning to the text.<br /><br /><code>"bold"</code>: Set the text in bold. Combine with an “accessibilyRole“ of “strong“ to add more meaning to the text. |
```

File: packages/checkout-ui-extensions/src/components/TextBlock/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ optional = ?
...
-| emphasis? | <code>"stress" &#124; "offset" &#124; "strong"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"stress"</code>: Indicate emphatic stress. By default the text will be italic and render a `<em>` tag in an HTML host.<br /><br /><code>"offset"</code>: Indicate an offset from the normal prose of the text. Typically used to indicate a foreign word, fictional character thoughts, or when the text refers to the definition of a word instead of representing its semantic meaning. By default the text will be italic and render a `<i>` tag in an HTML host.<br /><br /><code>"strong"</code>: Indicate strong importance, seriousness, or urgency. By default the text will be bold and render a `<strong>` tag in an HTML host. |
+| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic. Combine with an accessibilyRole of 'offset' or 'stress' to add more meaning to the text.<br /><br /><code>"bold"</code>: Set the text in bold. |
```
Comment (jffortier) [inline packages/checkout-ui-extensions/src/components/TextBlock/README.md:11]:
```suggestion
| emphasis? | <code>"italic" &#124; "bold"</code> | Use to emphasize a word or a group of words compared to other nearby text.<br /><br /><code>"italic"</code>: Set the text in italic.<br /><br /><code>"bold"</code>: Set the text in bold. |
```

**Review by jffortier — APPROVED** [review]
Small changes (my bad, sorry). I'll bring them back to main as well.

File: packages/checkout-ui-extensions/src/components/TextBlock/README.md (hunk 2 above)
Comment (jamesvidler) [reply packages/checkout-ui-extensions/src/components/TextBlock/README.md:11]: @jffortier the `Emphasis` is a shared type so the descriptions for both `TextBlock` and `Text` would need to be same no?

Comment (jamesvidler) [reply packages/checkout-ui-extensions/src/components/TextBlock/README.md:11]: Ya, the `TextBlockProps` don't include the `accessibilityRole` so you might need to adjust the description for `italic` and `bold` to not mention `accessibilityRole`.

File: packages/checkout-ui-extensions/src/components/Text/README.md (hunk 1 above)
Comment (igor10k) [reply packages/checkout-ui-extensions/src/components/Text/README.md:12]: Shouldn't we wrap accessibilyRole, offset and stress in backticks (`) to make them render like code?

File: packages/checkout-ui-extensions/src/components/TextBlock/README.md (hunk 2 above)
Comment (jamesvidler) [reply packages/checkout-ui-extensions/src/components/TextBlock/README.md:11]: Yep, it gets pulled into `shopify.dev` as well. If you open a PR in `checkout-web` with the ts-doc changes I can bring that update in and regen the docs with it.

**Pull request merged**